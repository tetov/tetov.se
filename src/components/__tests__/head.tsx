import React from "react";
import renderer from "react-test-renderer";
import HeadComponent from "src/components/head";

describe("HeadComponent", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <HeadComponent
          description="The dead trees waited to be ignited by the smallest spark and seek their revenge."
          imageUrl="http://runte.org/velit-porro-sapiente-id-aut-error-quis.png"
          pageTitle="We have a lot of rain in June."
          pathname="we-have-a-lot"
        >
          <meta property="og:type" content="article" id="og:type" />
          <meta
            property="og:article:published_time"
            content="2022-03-22T12:57:14Z"
          />
        </HeadComponent>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
