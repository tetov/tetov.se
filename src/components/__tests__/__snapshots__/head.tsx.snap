// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeadComponent renders correctly 1`] = `
Array [
  <title>
    We have a lot of rain in June. - Iðunn Gunda
  </title>,
  <meta
    content="We have a lot of rain in June."
    property="og:title"
  />,
  <meta
    content="Iðunn Gunda"
    property="og:site_name"
  />,
  <meta
    content="The dead trees waited to be ignited by the smallest spark and seek their revenge."
    id="description"
    name="description"
  />,
  <meta
    content="The dead trees waited to be ignited by the smallest spark and seek their revenge."
    id="description"
    property="og:description"
  />,
  <meta
    content="https://uiuc.eduhttp://runte.org/velit-porro-sapiente-id-aut-error-quis.png"
    property="og:image"
  />,
  <meta
    content="https://uiuc.eduwe-have-a-lot"
    property="og:url"
  />,
  <meta
    content="website"
    id="og:type"
    property="og:type"
  />,
  <meta
    content="summary"
    name="twitter:card"
  />,
  <meta
    content="@twttr"
    name="twitter:site"
  />,
  <meta
    content="@twttr"
    name="twitter:creator"
  />,
  <meta
    content="article"
    id="og:type"
    property="og:type"
  />,
  <meta
    content="2022-03-22T12:57:14Z"
    property="og:article:published_time"
  />,
]
`;
